src/jit/emitxarch.cpp

```diff
@@ -7785,6 +7813,32 @@ void emitter::emitDispIns(
             break;
         }
 
+        case IF_AWR_RRD_CNS:
+        case IF_MWR_RRD_CNS:
```

Comment:
These should be handled separately.

`AWR_RRD_CNS` should be using `emitDispAddrMode`
`MWR_RRD_CNS` should be using `emitDispClsVar`